### DISCUSSION TITLE
[Hover Card] What does `PositioningOptions` mean?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [chakra-ui/ark#2302] [Q&A] [answered]
### Description

The docs for https://ark-ui.com/docs/components/hover-card has a custom type called `PositioningOptions` but its type is not mentioned in the docs.

### Link to Reproduction (or Detailed Explanation)

https://ark-ui.com/docs/components/hover-card

### Steps to Reproduce

N/A

### Ark UI Version

N/A

### Framework

- [X] React
- [X] Solid
- [X] Vue

**Comment by segunadebayo** [comment, marked as answer]
Hi @tyteen4a03, 

The positioning options is used to customize the underlying positioning logic of the content relative to the trigger.

You can use it to control the placement, offset and much more.

Learn more here: https://github.com/chakra-ui/zag/blob/main/packages/utilities/popper/src/types.ts#L19

**Reply by tyteen4a03** [reply]
The issue is that I needed to look into the source code to see what the object's shape is. It was not included in the docs.

**Reply by segunadebayo** [reply]
Got it. Feel free to contribute to this part of the docs.
